Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1

```diff
@@ -92,6 +93,25 @@ function Write-DTLServiceDeprecationMessageIfRequired
     }
 }
 
+function Publish-Azure-Telemetry
```

Comment:
Remove the - between Azure and Telemetry.

Reply:
Done.